<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableViewWidgetClass</class>
 <widget class="QWidget" name="TableViewWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TableViewWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ATableView" name="table_view"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="push_btn_insert">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>insert</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="push_btn_remove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>remove</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="push_btn_carry">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ATableView</class>
   <extends>QTableView</extends>
   <header>atableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
